circ/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
I wish circ to be released as a separate cargo package (at least). to be able to publish it as its own (separating from smr-benchmark). maybe we can make circ (and cdrc-rs) as a cargo package inside this same repo.